content/en/docs/contribute/style/style-guide.md

```diff
@@ -380,6 +380,47 @@ The output is:
 
 {{< kat-button >}}
 
+### Downloadable code blocks
+
+Use {{</* codenew */>}} to make code samples (manifests) downloadable.
+
+ A way to include code in your topic is to create a new, complete sample file (or group of sample files) and then reference the sample from your topic. Use this method to include sample YAML files when the sample is generic and reusable, and you want the reader to try it out themselves.
+
+```none
+{{</* codenew file="application/deployment.yaml" */>}}
+```
+
+The output is:
+
+{{< codenew file="application/deployment.yaml" >}}
+
+Here is an example page containing that shortcode:
+
+https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/
+
+When adding a new standalone sample file, such as a YAML file, place the code in one of the `<LANG>/examples/` subdirectories where `<LANG>` is the language for the topic. In your topic file, use the codenew shortcode:
+
+```none
+{{</* codenew file="<RELPATH>/my-example-yaml>" */>}}
```

Comment:
hello @anshprat .Thanks for the updates. It looks as if this content is created from:
https://kubernetes.io/docs/contribute/style/write-new-topic/#including-code-from-another-file.
Personally, I like that the content appears in the `write-new-topic` page.
You could include a very brief section in the main style guide page about the `codenew` shortcode and link to the `write-new-topic` page, for example:

```
## Downloadable code blocks

Use the [codenew shortcode](/docs/contribute/style/write-new-topic/#including-code-from-another-file) to create downloadable code samples (manifests).
```
You can add any additional content (lines 415 - 423) to the existing section in the `write-new-topic` page.

Reply:
Got it, will update.